bluebanquise.spec

```diff
@@ -115,4 +119,5 @@ end}
 
 %changelog
 * Sun Feb  2 2020 Bruno Travouillon <devel@travouillon.fr>
+* Fri Apr 14 2020 strus38 <indigoping4cgmi@gmail.com>
```

Comment:
Almost sure today is not Friday. Can't wait for the week-end? :D

Reply:
fixed